StellarApp/Controllers/SolarViewController.swift

```diff
@@ -1,75 +1,242 @@
-//
-//  ViewController.swift
-//  StellarApp
-//
-//  Created by TingxinLi on 4/5/19.
-//  Copyright © 2019 TingxinLi. All rights reserved.
-//
-
 import UIKit
 import SceneKit
 import ARKit
 
+enum PlayAnimation {
+    case regular
+    case animation
+}
+
+enum PortalChange {
+    case reality
+    case galaxy
+}
+
 class SolarViewController: UIViewController, ARSCNViewDelegate {
+    
 
     @IBOutlet var sceneView: ARSCNView!
+
+    
+    let centerNode = CenterNode.getCenterNode()
+    var playAnimation: PlayAnimation = .animation
+    var portalChange: PortalChange = .reality
+   
+    
+    let solarView = SolarView()
     
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        // Set the view's delegate
         sceneView.delegate = self
         
-        // Show statistics such as fps and timing information
         sceneView.showsStatistics = true
         
-        // Create a new scene
-        let scene = SCNScene(named: "art.scnassets/ship.scn")!
+        view.addSubview(solarView)
         
-        // Set the scene to the view
-        sceneView.scene = scene
+
+        
+        sceneView.scene.rootNode.addChildNode(centerNode)
+        Planet.getPlanets().forEach { centerNode.addChildNode($0) }
+        
+        Sound.playSound(sound: "background", format: "mp3")
+
+        solarView.playButton.addTarget(self, action: #selector(playPressed), for: .touchUpInside)
+        solarView.mySwitch.addTarget(self, action: #selector(portalSwitch), for: .valueChanged)
+       
+        
+        
+//        let tapGestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(tapped))
+//        let pinchGestureRecognizer = UIPinchGestureRecognizer(target: self, action: #selector(pinch))
+//
+//        sceneView.addGestureRecognizer(tapGestureRecognizer)
+//        sceneView.addGestureRecognizer(pinchGestureRecognizer)
```

Comment:
lots of commented code here. I think you can delete it.

Reply:
got it